crates/turborepo-lib/Cargo.toml

```diff
@@ -26,26 +27,46 @@ axum-server = "0.4.4"
 chrono = { workspace = true, features = ["serde"] }
 clap = { workspace = true, features = ["derive"] }
 clap_complete = { workspace = true }
+command-group = { version = "2.1.0", features = ["with-tokio"] }
 config = "0.13"
 console = { workspace = true }
 dialoguer = { workspace = true, features = ["fuzzy-select"] }
 dirs-next = "2.0.0"
 dunce = { workspace = true }
 env_logger = "0.10.0"
+futures = "0.3.26"
 glob-match = "0.2.1"
+hex = "0.4.3"
 hostname = "0.3.1"
+humantime = "2.1.0"
 indicatif = { workspace = true }
 lazy_static = { workspace = true }
 log = { workspace = true }
-predicates = { workspace = true }
+notify = "5.1.0"
+pidlock = { git = "https://github.com/arlyon/pidlock", branch = "master" }
```

Comment:
I think we need to vendor and review this if we want to use it.

Reply:
I am waiting for some PRs but I will vendor for now and we can consider breaking it out again once they merge. (https://github.com/rockstar/pidlock/pulls)